src/drive/mobile/lib/i18n.js

```diff
@@ -0,0 +1,12 @@
+import { getLang } from './init'
+import { _polyglot, initTranslation } from 'cozy-ui/react/I18n/translation'
+
+// used when translating things outside of components, eg. native UIs
+export const getTranslateFunction = () => {
```

Comment:
Later we may choose to put this code inside `cozy-ui/react/I18n/translation`, don't we?

Reply:
Yeah maybe, but then it would require `lang` and `dictRequire ` as arguments, which would make it nearly the same as `initTranslation`. But maybe we can find an elegant way to do this.